qutebrowser/browser/commands.py

```diff
@@ -1748,3 +1756,19 @@ def fullscreen(self, leave=False, enter=False):
 
         log.misc.debug('state before fullscreen: {}'.format(
             debug.qflags_key(Qt, window.state_before_fullscreen)))
+
+    def _possibly_open_in_existing_tab(self, url):
+        if not config.val.tabs.switch_to_tab_on_url_match:
+            return False
+
+        tuple = utils.tuple_for_url_in_tab(url.toDisplayString())
+
+        if tuple is not None:
+            index = tuple[2]
```

Comment:
You call this `index`, but in `tuple_for_url_in_tab` is returning the `title` for the second field.

Reply:
It was indeed the index. But the function now just returns it, instead of the whole tuple.